app/controllers/posts_controller.rb

```diff
@@ -37,6 +38,32 @@ def index
 
   def homepage
     @latest_post = Post.order(sticky: :desc, created_at: :desc).first
+    @preview = params[:preview] == "1" && current_user.can_administrate_livestream?
+  end
+
+  def livestream_management
+  end
+
+  def update_test_link
+    new_value = params[:new_test_value]
+    test = ServerSetting.find_or_create_by(name: "TEST_wc2025_video_url")
```

Comment:
Can the string values like `TEST_wc2025_video_url` (and others) be stored as constants `WC2025_VIDEO_URL` somewhere? In case of doubt, putting them under `ServerSetting` directly shall be fine, just so that you avoid typos in your numerous `find` or `find_by` invocations.

Reply:
Done (pending discussion)